docs/tutorial/part-four/index.md

```diff
@@ -1,106 +1,106 @@
 ---
-title: Data in Gatsby
+title: Daten in Gatsby
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Welcome to Part Four of the tutorial! Halfway through! Hope things are starting
-to feel pretty comfortable 😀
+Willkommen zum vierten Teil des Tutorials! Zur Hälfte fertig! Hoffentlich fangen
+die Dinge an vertrauter zu werden 😀
 
-## Recap of the first half of the tutorial
+## Rückblick auf die erste Hälfte des Tutorials
 
-So far, you've been learning how to use React.js—how powerful it is to be able to
-create your _own_ components to act as custom building blocks for websites.
+Bislang hast du gelernt React.js einzusetzen und wie mächtig es ist, *eigene* Komponenten
+schreiben zu können, die als benutzerdefinierte Bausteine für Websites agieren.
 
-You’ve also explored styling components using CSS Modules.
+Du has auch mit Hilfe der CSS Modules, Styling Components erkundet.
```

Comment:
I'm not sure about translating "CSS Modules" to "CSS Modulen", because if I'd expect someone to search for this, "CSS Modules" would trigger more targeted results. Btw I tried to rephrase to something like:

> Mit Hilfe der CSS Modules hast du auch Styling Components erkunden können.

In my opinion this is more understandable and prevents that the reader gets confused by "...von CSS Modulen Styling Components"

Reply:
I agree!